fix port number typo.

takekazuomi opened the pull request:
MSI Agent default port number is 50342. not 50432.

You can check the port with the following command.
```
$ sudo lsof -i | grep msi
msi-exten  1758         root    6u  IPv4  21079      0t0  TCP localhost:50342 (LISTEN)
```
or  logs in
/var/log/azure/Microsoft.ManagedIdentity.ManagedIdentityExtensionForLinux/

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

arramac commented:
ack

takekazuomi committed:
fix port number typo.

MSI Agent default port number is 50342. not 50432.
You can check the port with the following command.

```
$ sudo lsof -i | grep msi
```
or  logs in
/var/log/azure/Microsoft.ManagedIdentity.ManagedIdentityExtensionForLinux/
